src/backend/aspen/workflows/nextstrain_run/create_phyloruns.py

```diff
@@ -150,7 +150,7 @@ def retry_template_args_for_focal_group(
     for level in location_hierarchy:
         if getattr(location, level):
             current_location_level.append(level)
-    start_date_attempts = ["6 months ago", "12 months ago"]
+    start_date_attempts = ["6 months ago", "24 months ago"]
```

Comment:
should we try 6mo then 12mo then 24mo? or go straight from 6mo to 2yrs?

Reply:
good point~~